tests/core/data_layer/test_data_rpc.py

```diff
@@ -116,7 +116,7 @@ async def farm_block_with_spend(
 
 
 def check_mempool_spend_count(full_node_api: FullNodeSimulator, num_of_spends: int) -> bool:
-    return len(full_node_api.full_node.mempool_manager.mempool.sorted_spends) == num_of_spends
```

Comment:
note how the current version counts the number of *sorted_spends*, which means counting the number of fee rate *levels*, not the number of items in the mempool. The patched version does away with easy access to the fee rate levels and instead count the actual number of mempool items. This also requires a few tests to change their numbers

Reply:
Yes, that old variable name was misleading.